optimade/adapters/structures/ase.py

```diff
@@ -7,7 +7,6 @@
 
 For more information on the ASE code see [their documentation](https://wiki.fysik.dtu.dk/ase/).
 """
-from typing import Dict
 
```

Comment:
```suggestion
```

Reply:
Again, if black is happy here I am happy...